pom.xml

```diff
@@ -602,6 +657,107 @@
                 </plugins>
             </build>
         </profile>
+
```

Comment:
Tested this profile on my local VM.

Reply:
![workspace one connectors](https://user-images.githubusercontent.com/5927317/38615718-499adfd8-3daf-11e8-84bd-6c14d49228b9.png)

Reply:
![bitbucket server connector](https://user-images.githubusercontent.com/5927317/38616890-28d7d0a4-3db3-11e8-8df1-a75123abe8fd.png)